values.yaml

```diff
@@ -0,0 +1,9 @@
+---
+images:
+  bash: registry.access.redhat.com/ubi8-minimal:latest
+  openshift-cli: registry.redhat.io/openshift-pipelines/pipelines-cli-tkn-rhel8
```

Comment:
This is the `tkn` image (and probably not the public address), not the `oc` image.

Reply:
I have updated the image for openshift-client.